packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmDictionaryTests.kt

```diff
@@ -573,6 +628,14 @@ internal interface DictionaryApiTester<T, Container> : ErrorCatcher {
     fun entries_remove()
     fun entries_removeAll()
     fun entry_equals()
+    fun values_addTrows()
+    fun values_clear()
+    fun values_iteratorNext() // This tests also hasNext
+    fun values_iteratorRemove() // This tests also hasNext
+    fun values_iteratorConcurrentModification()
+    fun values_remove()
+    fun values_removeAll()
+    fun values_retainAll()
```

Comment:
Should we also add an `equals` test for the values set?

Reply:
I will also add one for the entry set as well, though observe the restrictions on byte arrays will apply until dealt with in the upcoming PR.